concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/30 Work with Tables.md

```diff
@@ -0,0 +1,87 @@
+Users can insert and work with tables if you add the following items to the toolbar:
```

Comment:
'Insert' is a particular case of 'work'. Maybe 'create and customize/modify tables'?

Reply:
```suggestion
Users can insert and modify tables if you add the following items to the toolbar:
```